analytic_engine/src/row_iter/record_batch_stream.rs

```diff
@@ -77,83 +101,76 @@ pub type SequencedRecordBatchStream = Box<
         + Unpin,
 >;
 
-/// Filter the `sequenced_record_batch` according to the `filter` if necessary.
-/// Returns the original batch if only a small proportion of the batch is
-/// filtered out.
-/// The `selected_rows_buf` is for reuse.
+/// Filter the `sequenced_record_batch` according to the `predicate` if
+/// necessary. Returns the original batch if only a small proportion of the
+/// batch is filtered out.
 fn maybe_filter_record_batch(
     mut sequenced_record_batch: SequencedRecordBatch,
-    filter: &RecordBatchFilter,
```

Comment:
The [file](https://github.com/CeresDB/ceresdb/blob/main/table_engine/src/predicate/filter_record_batch.rs) defining this `RecordBatchFilter` may be useless and we can remove it now.